pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -147,6 +151,24 @@ func GatherMostRecentMetrics(i *Gatherer) func() ([]record.Record, []error) {
 			klog.Errorf("Unable to retrieve most recent metrics: %v", err)
 			return []record.Record{{Name: "config/metrics", Item: RawByte(fmt.Sprintf("# error: %v\n", err))}}, nil
 		}
+
+		rsp, err := i.metricsClient.Get().AbsPath("federate").
+			Param("match[]", "ALERTS").
+			Stream()
+		if err != nil {
+			// write metrics errors to the file format as a comment
+			klog.Errorf("Unable to retrieve most recent alerts from metrics: %v", err)
+			return []record.Record{{Name: "config/metrics", Item: RawByte(fmt.Sprintf("# error: %v\n", err))}}, nil
+		}
+		r := NewLineLimitReader(rsp, metricsAlertsLinesLimit)
+		alerts, err := ioutil.ReadAll(r)
+		if err != nil {
+			klog.Errorf("Unable to read most recent alerts from metrics: %v", err)
+			return nil, []error{err}
+		}
+
+		data = append(data, []byte("# Alerts \n")...)
```

Comment:
shouldn't this be uppercase?

Reply:
I'm not really sure because I haven't worked with this part of the collected data before. Should it?

Reply:
What was there before this change (coming from prometheus directly):

```
# TYPE ALERTS untyped
```

Reply:
the Prometheus parser in Python reads that line, if something does not quite match it may not work, so keep what Ivan mentioned

Reply:
@iNecas @alexandrevicenzi I believe this change doesn't actually touch that at all. That line will still be there. It will just be prefixed by this line that's been added by Martin.
If that's invalid and if that would cause trouble with the parser, then please let me know where should I put the total number of alerts because there doesn't seem to be any other suitable place for it.